templates/template_data.go

```diff
@@ -57,6 +64,30 @@ type ExtendedData struct {
 	ExternalURL string `json:"externalURL"`
 }
 
+// FromContent calls Parse on all provided template content and returns the resulting Template. Content equivalent to templates.FromGlobs.
+func FromContent(templateContents []string, options ...template.Option) (*Template, error) {
```

Comment:
I think we can use `tmpls` as it's become idiomatic to mean template text (or slice of template texts):

```suggestion
func FromContent(tmpls []string, options ...template.Option) (*Template, error) {
```